appservice/src/SiteWrapper.ts

```diff
@@ -346,6 +417,75 @@ export class SiteWrapper {
         await this.waitForDeploymentToComplete(kuduClient, outputChannel);
     }
 
+    private async getJsonRequest(url: string, token: string, node: IAzureNode): Promise<Object[]> {
+        // Reference for GitHub REST routes
+        // https://developer.github.com/v3/
+        // Note: blank after user implies look up authorized user
+        try {
+            const gitHubResponse: string = await requestP.get(url, {
+                headers: {
+                    Authorization: `token ${token}`,
```

Comment:
Let's use `new TokenCredentials(token)` and the signRequest method from the PR I just merged